index.html

```diff
@@ -13,14 +13,19 @@
       rel="stylesheet"
     />
     <link rel="stylesheet" href="style.css" />
-    <title>Raihan's Portfolio</title>
+    <title>Ibrahim's Portfolio</title>
   </head>
 
   <body>
     <header class="header">
       <nav class="header_nav">
         <div class="header_logo blue"><a class="blue" href="#">My Logo</a></div>
         <div class="header_menu">
```

Comment:
- [ ] The `mobile version` of your portfolio appears to be affected. Kindly try to fix this issues.
     - The content does not fit and you have to use the horizontal scroll bar to see everything
     - The general `font-size` is larger than the original design
     
![image](https://user-images.githubusercontent.com/98453979/199769597-d172eb36-87a1-4803-be41-fd7b358a78f2.png)

![image](https://user-images.githubusercontent.com/98453979/199769822-e4277c67-cfdd-49c8-adad-04aeda8d2bcc.png)


### Checkout Figma

![image](https://user-images.githubusercontent.com/98453979/199770057-f4833a61-3fe9-4dbd-94cf-02787fb8ebd4.png)

![image](https://user-images.githubusercontent.com/98453979/199770245-b525ccc8-b373-4a13-b60b-06f87b4e5b75.png)

Reply:
the horizontal scroll was due to zoom

Reply:
- [ ] :red_circle: I think the previous reviewer was right. I am still seeing the horizontal scroll on The mobile version of your portfolio Please make sure you try to fix this issue as it makes the mobile version of your portfolio unresponsive.